sentry-core/src/main/java/io/sentry/core/ISentryClient.java

```diff
@@ -25,18 +25,23 @@ default SentryId captureEvent(SentryEvent event, @Nullable Object hint) {
     return captureEvent(event, null, hint);
   }
 
-  default SentryId captureMessage(String message) {
-    return captureMessage(message, null);
-  }
-
-  default SentryId captureMessage(String message, @Nullable Scope scope) {
+  default SentryId captureMessage(String message, SentryLevel level, @Nullable Scope scope) {
     SentryEvent event = new SentryEvent();
     Message sentryMessage = new Message();
     sentryMessage.setFormatted(message);
     event.setMessage(sentryMessage);
+
+    if (scope == null || scope.getLevel() == null) {
```

Comment:
The default should be just dumb helper to what a user would do which is create an event and set the level.
The scope dealing is stuff happening under the hoods (MainEventProcessor) and if we were to change that decision (scope > event for levels) we would change in 1 place,